src/main/webapp/app/exercises/shared/statistics/exercise-statistics.component.html

```diff
@@ -0,0 +1,34 @@
+<div *ngIf="exercise">
+    <div class="row mb-5">
+        <h2 class="col-md-2" id="statistics-page-heading" jhiTranslate="exercise-statistics.title">Course statistics</h2>
+        <div class="col-md-9 btn-group btn-group-toggle" data-toggle="buttons">
+            <label class="btn btn-secondary" [class.active]="currentSpan === SpanType.DAY">
+                <input type="radio" name="options" id="option1" (click)="onTabChanged(SpanType.DAY)" />{{ 'statistics.span.day' | artemisTranslate }}
+            </label>
+            <label class="btn btn-secondary" [class.active]="currentSpan === SpanType.WEEK">
+                <input type="radio" name="options" id="option2" checked (click)="onTabChanged(SpanType.WEEK)" />{{ 'statistics.span.week' | artemisTranslate }}
+            </label>
+            <label class="btn btn-secondary" [class.active]="currentSpan === SpanType.MONTH">
+                <input type="radio" name="options" id="option3" (click)="onTabChanged(SpanType.MONTH)" />{{ 'statistics.span.month' | artemisTranslate }}
+            </label>
+            <label class="btn btn-secondary" [class.active]="currentSpan === SpanType.QUARTER">
+                <input type="radio" name="options" id="option4" (click)="onTabChanged(SpanType.QUARTER)" />{{ 'statistics.span.quarter' | artemisTranslate }}
+            </label>
+            <label class="btn btn-secondary" [class.active]="currentSpan === SpanType.YEAR">
+                <input type="radio" name="options" id="option5" (click)="onTabChanged(SpanType.YEAR)" />{{ 'statistics.span.year' | artemisTranslate }}
+            </label>
+        </div>
+    </div>
+    <div class="mb-3">
+        <jhi-exercise-detail-statistics [exercise]="exercise" [doughnutStats]="exerciseStatistics"></jhi-exercise-detail-statistics>
+    </div>
+    <jhi-statistics-score-distribution-graph
+        *ngIf="exerciseStatistics"
+        [averageScoreOfExercise]="exerciseStatistics?.averageScoreOfExercise"
+        [scoreDistribution]="exerciseStatistics?.scoreDistribution"
+        [numberOfExerciseScores]="exerciseStatistics?.numberOfExerciseScores"
+    ></jhi-statistics-score-distribution-graph>
+    <div *ngFor="let graph of graphTypes; let i = index" id="graph-{{ i }}" class="graph">
```

Comment:
Id is a bit too generic, can be a problem in the future.

Reply:
👍